### PR TITLE
Add pending notifications incident history messages

### PR DESCRIPTION
**Pull request by yhabteab**

### CODE REVIEW

**Comment by julianbrost** [comment]
Note that even though I already merged https://github.com/Icinga/icinga-notifications/pull/99, I didn't really test this PR because the current state doesn't work (presumably due to the change channel references to IDs change) and I can't rebase without conflicts (don't look too bad, but I didn't bother resolving them).

**Review by nilmerg — CHANGES_REQUESTED** [review]
The state must be translatable. The value in the database is not a label.

File: library/Notifications/Model/IncidentHistory.php [hunk 1]
```diff
@@ -84,4 +91,18 @@ public function createRelations(Relations $relations)
         $relations->belongsTo('rule_escalation', RuleEscalation::class)->setJoinType('LEFT');
         $relations->belongsTo('channel', Channel::class)->setJoinType('LEFT');
     }
+
+    public static function translateNotificationState(string $state): string
```
Comment (nilmerg) [inline library/Notifications/Model/IncidentHistory.php:95]: Add documentation

File: library/Notifications/Model/IncidentHistory.php [hunk 2]
```diff
@@ -84,4 +91,18 @@ public function createRelations(Relations $relations)
...
+    {
+        switch ($state) {
+            case 'sent':
+                return t('sent', 'contact.notifications.state');
```
Comment (nilmerg) [inline library/Notifications/Model/IncidentHistory.php:99]: *notifications.transmission.state* fits better IMHO

**Review by nilmerg — CHANGES_REQUESTED** [review]

File: library/Notifications/Model/IncidentHistory.php (hunk 1 above)
Comment (nilmerg) [reply library/Notifications/Model/IncidentHistory.php:95]: A docblock is obligatory.

**Review by nilmerg — APPROVED** [review]

**Review by nilmerg — CHANGES_REQUESTED** [review]
wait, there are conflicts

**Review by nilmerg — APPROVED** [review]

**Pull request merged**